nepal_compliance/nepal_compliance/report/sales_register_ird/sales_register_ird.py

```
@@ -95,13 +102,18 @@ def get_data(filters):
                     taxable_domestic_nc += amt
 
         total_taxable = taxable_domestic_nc + taxable_import_nc + capital_taxable_amount
-        tax_domestic_nc = taxable_domestic_nc * 0.13 if total_taxable else 0
+        total_tax = flt(inv.total_tax)
+
+        if total_tax == 0 or total_taxable == 0:
+            tax_domestic_nc = tax_import_nc = tax_capital = 0
```

Ruff (0.12.2)

Local variable `tax_import_nc` is assigned to but never used

Remove assignment to unused variable `tax_import_nc`

(F841)